### PR TITLE
Adding `disallowInjectedContainerAccess` to warn about Ember 2.3 deprecations.

### LINKED ISSUE #4: Ember 2.3.0 Deprecation Checklist
- [ ] [#container access in instances](http://emberjs.com/deprecations/v2.x/#toc_injected-container-access)

### PR DESCRIPTION
**Pull request by workmanw**
Also resolves #4.

This is my first time ever working with JSCS or ESTrees so it's very possible there is a better way to handle this. Willing to take any feedback.

Also, it seems counter intuitive to start with the Ember 2.3 deprecation (instead of the outstanding Ember 2.2), but that one seems very hard, possibly impossible. I will comment further on that story.

### CODE REVIEW

**Comment by minichate** [comment]
This looks ahead!

**Comment by minichate** [comment]
Argh. Typo :) "this looks great!"

File: test/lib/rules/disallow-injectedcontaineraccess.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    var checker = global.checker({
+        plugins: ['./lib/index']
+    });
+
+    describe('not configured', function() {
+
+      it('should report with undefined', function() {
+        global.expect(function() {
+          checker.configure({disallowInjectedContainerAccess: undefined});
+        }).to.throws(/requires a true value/i);
```
Comment (minichate) [inline test/lib/rules/disallow-injectedcontaineraccess.js]: It'd be great to change these messages to `requires a boolean value` to match conventions. See https://github.com/minichate/jscs-ember-deprecations/blob/master/test/lib/rules/disallow-defaultlayout.js#L11 as an example.

**Comment by minichate** [comment]
I'm happy merging this as-is if you'd like; or if you feel like fixing https://github.com/minichate/jscs-ember-deprecations/pull/33#discussion_r50935987 I can wait

**Comment by workmanw** [comment]
> It'd be great to change these messages to requires a boolean value to match conventions.

Yeap, Agreed. Change made.

Thank you for this repo! It's be a huge help for us :)

**Comment by minichate** [comment]
Shipped in `0.0.27`

**Pull request merged**